src/state/slices/foxEthSlice/foxEthSlice.ts

```diff
@@ -91,8 +90,8 @@ export const foxEth = createSlice({
       if (!draftState[action.payload.accountAddress ?? '']) {
         draftState[action.payload.accountAddress ?? ''] = {
           farmingOpportunities: [],
-          lpOpportunity: {} as EarnOpportunityType,
```

Comment:
This was our issue. Down the line we'd attempt to access properties of `EarnOpportunityType`, but of course, couldn't.